src/Lykke.Sdk/LykkeStarter.cs

```diff
@@ -13,9 +12,26 @@ namespace Lykke.Sdk
     [PublicAPI]
     public static class LykkeStarter
     {
+        private static object _locker = new object();
+        private static IWebHostFactory _webHostBuilderFactory = new WebHostFactory();
+
         /// <summary>DEBUG/RELEASE mode flag.</summary>
         public static bool IsDebug { get; private set; }
 
+        /// <summary>WebHostFactory for creating IWebHostBuilder.</summary>
+        public static IWebHostFactory WebHostFactory
+        {
+            set
+            {
+                lock (_locker)
+                {
+                    //Ignores null
+                    if (value != null)
+                        _webHostBuilderFactory = value;
```

Comment:
I'ts better to use thrown ArgumentNullException here using ?? operator

Reply:
Agree

Reply:
Ok